### DISCUSSION TITLE
Hyprland not working after update it

### DISCUSSION THREAD

**Discussion by mshzsh** [hyprwm/Hyprland#814] [Q&A] [answered]
`dbus-run-session Hyprland :`

[log.txt](https://github.com/hyprwm/Hyprland/files/9739674/log.txt)

**Comment by mshzsh** [comment]
`cat /tmp/hypr/$(ls -t /tmp/hypr/ | head -n 1)/hyprland.log :`

[log2.txt](https://github.com/hyprwm/Hyprland/files/9739698/log2.txt)

**Comment by vaxerski** [comment]
"after update"
from what to what

**Reply by mshzsh** [reply]
from hyprland-git v14.*(I dont remember exactly) to v15.1 (now)

**Reply by vaxerski** [reply]
oh god. Give me the coredump then (see issue guidelines in docs/)

**Reply by mshzsh** [reply]
coredump file is 1G!

**Reply by vaxerski** [reply]
thats why we have the issue guidelines that tell you what you should provide.

**Comment by mshzsh** [comment, marked as answer]
coredumpctl info 430 (Hyprland PID) output : 
`No coredumps found`

after login with `ly` or run in `tty` show following screen : 


![hyprland](https://user-images.githubusercontent.com/41986782/194758192-3b6356fe-1ddf-4ca8-911e-0e28d460b02d.jpg)

What solution do you suggest?

**Reply by vaxerski** [reply]
are you sure it's the correct PID that `coredumpctl` has shown you? Make sure to grab the LATEST, that it shows you (the one at the bottom after you press END)

**Reply by mshzsh** [reply]
Does it matter if use `coredumpctl` in `tty` or on `bspwm`?

**Reply by mshzsh** [reply]
[coredump.txt](https://github.com/hyprwm/Hyprland/files/9741724/coredump.txt)

**Reply by vaxerski** [reply]
bruh thats not a hyprland coredump

**Reply by mshzsh** [reply]
[hyprland_coredump.txt](https://github.com/hyprwm/Hyprland/files/9741845/hyprland_coredump.txt)

**Reply by vaxerski** [reply]
super odd. Can you try `misc:disable_hyprland_logo=yes`?

**Reply by mshzsh** [reply]
I did but it didn't work

```
misc {
    disable_hyprland_logo=yes
}
```

**Reply by vaxerski** [reply]
you'll have to narrow it down then. Can you check: 0.15, 0.15.1 and HEAD?

**Reply by mshzsh** [reply]
I completely remove hyprland with caches and then installed it again (hyprland-git AUR)  and now it worked!
Hoping one day to have hyprland on my laptop with dual monitors  :smiley:

**Reply by vaxerski** [reply]
:) glad it works!

**Reply by Si-Ki** [reply]
I had the same error, added this env var to startw file, `export LIBSEAT_BACKEND=logind` and it worked fine.

**Reply by thearpankumar** [reply]
I still have the issue after the latest update of manjaro
